app/services/IntersectService.ts

```diff
@@ -1,28 +1,14 @@
 import { checkIntersect as intersect } from '../helpers/Intersect';
 import { HealthcareAuthority } from '../store/types';
 import { emitGPSExposureInfo } from '../gps/exposureInfo';
-import { INTERSECT_INTERVAL } from '../constants/history';
-import BackgroundFetch from 'react-native-background-fetch';
-
-const bgTaskConfig = {
-  minimumFetchInterval: INTERSECT_INTERVAL,
-  // Android options
-  forceAlarmManager: false, // <-- Set true to bypass JobScheduler.
-  stopOnTerminate: false,
-  startOnBoot: true,
-  requiredNetworkType: BackgroundFetch.NETWORK_TYPE_NONE, // Default
-  requiresCharging: false, // Default
-  requiresDeviceIdle: false, // Default
-  requiresBatteryNotLow: false, // Default
-  requiresStorageNotLow: false, // Default
-};
+import BackgroundTaskService from './BackgroundTaskService';
 
 class IntersectService {
   isServiceRunning = false;
   private nextJob: HealthcareAuthority[] | null = null;
 
   constructor() {
-    this.startBackgroundTask();
+    BackgroundTaskService.start();
```

Comment:
This seems weird because of how they use each other -- 
can `BackgroundTaskService` be started from somewhere central in the app? Maybe App.js?

Reply:
sure thing, want me to change this now or can we do it as part of the backfill work?